Kbs.Business/Game/GameValidator.cs

```diff
@@ -22,4 +23,25 @@ public Dictionary<string, string> ValidateForCreate(GameEntity game)
 
         return result;
     }
+    
+    public Dictionary<string, string> ValidateForUpdate(GameEntity game)
+    {
+        var result = new Dictionary<string, string>();
+        if (string.IsNullOrWhiteSpace(game.Name))
+        {
+            result.Add(nameof(game.Name), "Naam is verplicht");
+        }
+
+        if (game.Date < DateTime.Now)
```

Comment:
Als een game voorbij is, alleen de naam moet aangepast worden, krijgt dit een error